src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -172,7 +175,9 @@ export class ChatService extends Disposable implements IChatService {
 				inputValue: transferredData.inputValue,
 				location: transferredData.location,
 				toolsAgentModeEnabled: transferredData.toolsAgentModeEnabled,
+				isWorkspaceTrusted: transferredData.isWorkspaceTrusted,
 			};
+			workspaceTrustManagementService.setWorkspaceTrust(transferredData.isWorkspaceTrusted);
```

Comment:
Shouldn't we still be asking the user whether they trust it?

Reply:
We're  trusting  empty folders created as a part of the /new workflow and do not want to show the trust dialog.
 Closed this pr since I wasnt happy with the fix. 
 
 Going this route: https://github.com/microsoft/vscode/pull/243288